src/modules/recipe/eventBubbling/__tests__/eventBubbling.test.js

```diff
@@ -0,0 +1,68 @@
+import { createElement } from 'lwc';
+import EventBubbling from 'recipe/eventBubbling';
+import { registerLdsTestWireAdapter } from '@salesforce/wire-service-jest-util';
+import getContactList from 'data/wireGetContactListProvider';
+
+const CONTACTS = [
+    {
+        Id: '0031700000pJRRSAA4',
+        Name: 'Amy Taylor',
+        Title: 'VP of Engineering',
+        Phone: '4152568563',
+        Email: 'amy@demo.net',
+        Picture: '/resources/images/demo/amy_taylor.jpg'
+    }
+];
+
+const getContactListWireAdapter = registerLdsTestWireAdapter(getContactList);
+
+describe('recipe-event-bubbling', () => {
+    afterEach(() => {
+        // The jsdom instance is shared across test cases in a single file so reset the DOM
+        while (document.body.firstChild) {
+            document.body.removeChild(document.body.firstChild);
+        }
+        // Prevent data saved on mocks from leaking between tests
+        jest.clearAllMocks();
+    });
+
+    it('renders contact list from wire', () => {
```

Comment:
I'd like to see overall alignment between the tests in `lwc-recipes` and `lwc-recipes-oss`.

Reply:
done.